test/xds/xds_client_integration_test.go

```diff
@@ -106,3 +107,43 @@ func (s) TestClientSideXDS(t *testing.T) {
 		t.Fatalf("rpc EmptyCall() failed: %v", err)
 	}
 }
+
+func (s) TestOutlierDetection(t *testing.T) {
```

Comment:
Could we please move this test to be in a file on its own. Otherwise over time, this file is going to become a dumping ground for tests. Thanks.

Reply:
Will do.